FirstProject/new-evaluation.html

```diff
@@ -89,26 +87,35 @@ <h4 style="color:white;">Senior</h4>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Should the candidate be hired?</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="The type of project that is suitable for the candidate
-Is guidance required for the candidate
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="The type of project that is suitable for the candidate-->
+<!--Is guidance required for the candidate-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">General Impression</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="*required
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="*required-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & Soft Skills</h2>
```

Comment:
Careful with these HTML entities! Browsers usually have good enough error recovery mechanisms that allow them to fix small issues such as this, but they can still lead to subtle, hard to track bugs.